cypress/integration/myworkflows.js

```diff
@@ -33,40 +33,6 @@ describe('Dockstore my workflows', function() {
                 .get('#refreshButton')
                 .should('not.have.class', 'disabled')
         });
-
-        it('Only Info and Labels tab should be enabled', function() {
```

Comment:
Just checking that this was deleted on purpose

Reply:
Yes since they are no longer ever disabled, we now just have a message saying to refresh.